src/vl-wizard.js

```diff
@@ -148,6 +157,25 @@ export class VlWizard extends vlElement(HTMLElement) {
   _previousPanesDisabledChangedCallback(oldValue, newValue) {
     this._panes.forEach((pane) => newValue != undefined ? pane.disablePreviousPane() : pane.enablePreviousPane());
   }
+
+  _observePanes(callback) {
+    const observer = new MutationObserver((mutations) => {
+      const hasPaneMutation = mutations.flatMap((mutation) => [...mutation.addedNodes]).some((node) => node instanceof VlWizardPane);
```

Comment:
hasNewPane eerder dan?

Reply:
ik zie redelijk veel die MutationObserver gebruikt worden de laatste tijd. daar zijn geen gevaren aan verbonden, cfr. waar er altijd gezegd is "pas op met te veel watchers in angular"?

Reply:
> hasNewPane eerder dan?

Kan ook zijn dat er een vl-wizard-pane verwijderd werd, vandaar `hasPaneMutation`.

> ik zie redelijk veel die MutationObserver gebruikt worden de laatste tijd. daar zijn geen gevaren aan verbonden, cfr. waar er altijd gezegd is "pas op met te veel watchers in angular"?

Zolang er geen mutations andere mutations veroorzaken en er een ketting aan watchers/observers en gevolgen ontstaat, verwacht ik geen problemen.

Reply:
maar er wordt alleen naar addedNodes gekeken? dan zou ik verwachten dat het alleen over toevoegingen gaat? of ik snap den API nie.

Reply:
Nee klopt, die worden gefilterd. Ik zal het aanpassen naar `hasNewPane`.